GAE/src/org/waterforpeople/mapping/dataexport/SurveySummaryExporter.java

```diff
@@ -234,17 +230,58 @@ protected SummaryModel buildDataModel(String surveyId, String serverBase, String
             }
             questionMap.put(group, questions);
         }
+        
+        if (loadOptionNames) {
```

Comment:
Given that this function doesn't take any extra parameters that are not present at the level of `loadAllQuestions()` I would suggest we extract it from here and avoid having to add an extra parameter for `loadAllQuestions()` i.e. 

```
export () {
   loadAllQuestions();
   if(...) { loadQuestionOptions(); }
}
```